mir-module/src/main/resources/xsl/metadata/mir-abstract.xsl

```diff
@@ -217,12 +217,29 @@
 
         <xsl:choose>
           <xsl:when test="count($abstracts/mods:abstract) &gt; 1">
-            <xsl:variable name="has-abstract-in-current-lang" select="$abstracts/mods:abstract[@xml:lang=$CurrentLang]"/>
+            <xsl:variable name="first-abstract-in-current-lang-node" select="$abstracts/mods:abstract[@xml:lang=$CurrentLang][1]"/>
+            <xsl:variable name="first-abstract-in-current-lang-position">
+              <xsl:for-each select="$abstracts/mods:abstract">
+                <xsl:sort select="@type"/>
+                <xsl:sort select="@xml:lang"/>
+
+                <xsl:if test=".= $first-abstract-in-current-lang-node">
+                  <xsl:value-of select="position()"/>
+                </xsl:if>
+              </xsl:for-each>
+            </xsl:variable>
+
             <div id="mir-abstract-tabs">
               <ul class="nav nav-tabs justify-content-end" role="tablist">
                 <xsl:for-each select="$abstracts/mods:abstract">
+                  <xsl:sort select="@type"/>
+                  <xsl:sort select="@xml:lang"/>
+
                   <xsl:variable name="tabName">
                     <xsl:choose>
+                      <xsl:when test="@type and $MIR.Layout.Abstract.Type.Classification">
+                        <xsl:value-of select="document(concat('classification:metadata:0:children:', $MIR.Layout.Abstract.Type.Classification,':',@type))//category/label[@xml:lang=$CurrentLang]/@text"/>
```

Comment:
could use `mcrxsl:getDisplayName('$MIR.Layout.Abstract.Type.Classification',@type)` instead of raw classification access. that method considers some edge cases